tdrs-backend/tdpservice/users/test/test_auth.py

```diff
@@ -147,7 +146,7 @@ class TestLoginAMS:
         "userinfo_endpoint": "http://openid-connect/userinfo"
     }
 
-    test_hhs_id = str(uuid.uuid4())
+    test_hhs_id = "1234567890"
```

Comment:
The `hhsid` field returned by AMS does not meet uuid requirements and instead appears to just be a large integer hence the change here and elsewhere.